### PR TITLE
Add Precis

### PR DESCRIPTION
**Pull request by leozqin**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

NOTE: I previously had [another PR](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/719) for this, but I've since force pushed to that branch so that PR may not be reopened.

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones. 
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/precis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# software name
```
Comment (kokomo123) [inline software/precis.yml:1]:
```suggestion
```
Feel free to omit this comment

**Review by kokomo123 — APPROVED** [review]
LGTM, no problems here! pending other thoughts from collaborators 👍🏻

File: software/precis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Precis
+website_url: "https://github.com/leozqin/precis"
+source_code_url: "https://github.com/leozqin/precis"
+description: "Extensibility-oriented RSS reader that can use LLMs (including local LLMs) to summarize rss entries with built-in notification support"
```
Comment (Ki-er) [inline software/precis.yml:4]:
```suggestion
description: "Extensibility-oriented RSS reader that can use LLMs (including local LLMs) to summarize RSS entries with built-in notification support"
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Consistent Description.

**Review by Rabenherz112 — COMMENTED** [review]
Submission would be a bit to early as the first release was May 20, which would make that release four months old on `2024-09-20`.

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**